themes/jquery.org/style.css

```diff
@@ -612,6 +612,14 @@ nav#main ul {
 		width: 100%;
 		text-align: center;
 	}
+
+	nav#main .searchform {
+		width: 100%;
+	}
+
+	nav#main ul {
+		width: 100%;
+	}
```

Comment:
Which previous block? Do you mean the very similar media query (1px diff) after this one?